src/chigame/api/serializers.py

```diff
@@ -1,9 +1,21 @@
 from rest_framework import serializers
 
-from chigame.games.models import Game
+from chigame.games.models import Game, User
 
 
 class GameSerializer(serializers.ModelSerializer):
     class Meta:
         model = Game
         fields = ("id", "name", "description", "min_players", "max_players")
+
+
+class UserListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ("name", "email")
```

Comment:
Please also add the primary key to users in the list view, that way someone can requests the list view, find an email of a user they want to reference, then follow the provided primary key to your already created `api/users/<int:pk>` view.

Reply:
This has been changed :) Will write a more descriptive description soon